app/views/unattended/provisioning_templates/snippet/puppet_setup.erb

```diff
@@ -59,7 +59,7 @@ $puppet_install_args = @(
   <%- if @host.puppet_ca_server.strip -%>
   "PUPPET_CA_SERVER=<%= @host.puppet_ca_server %>",
```

Comment:
It feels like this should be consistent. You've introduced a separate macro for the server, but the CA server is also determined via a similar mechanism.

Reply:
Is it ok, I fixed the if condition? `if string.strip` is always true, but maybe it was intentional?

Reply:
What I meant with consistent is that there should be a macro for the CA server. I wasn't talking about the strip part.

Reply:
I know, but I had a follow-up, I'll start a new thread next time for clarity.